components/listbox/listbox.ts

```diff
@@ -160,33 +160,23 @@ export class Listbox implements AfterContentInit,ControlValueAccessor {
     }
 
     onOptionClickMultiple(event, option) {
-        let metaKey = (event.metaKey || event.ctrlKey);
         let selected = this.isSelected(option);
-        let valueChanged = false;
 
         if (selected) {
-            if (metaKey) {
-                this.value.splice(this.findIndex(option), 1);
-            }
-            else {
-                this.value = [];
-                this.value.push(option.value);
-            }
-            valueChanged = true;
+            this.value.splice(this.findIndex(option), 1);
```

Comment:
Since it is multiple values, we do not want to clear all other items, we just want to remove the current item.